uktrade/data-hub-frontend
# Reactify companies local navigation bar

## Description of change

Reactifying the companies local navigation bar

## Test instructions

Visit a Company page e.g.
http://localhost:3000/companies/eea020f8-0d7c-416d-ac4c-38f4c888c2f0/activity
And you should be able to use the local nav bar to navigate to other pages as usual

## Screenshots

![image](https://user-images.githubusercontent.com/83657534/218511720-c0c3bee5-1929-482b-b29c-2a482ef93381.png)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
